pkg/apps/strategy/recreate/recreate.go

```diff
@@ -8,24 +8,24 @@ import (
 	"strings"
 	"time"
 
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/scale"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/client-go/util/retry"
 	kapi "k8s.io/kubernetes/pkg/apis/core"
-	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
-	kcoreclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
 
-	appsapi "github.com/openshift/origin/pkg/apps/apis/apps"
+	imageclientv1 "github.com/openshift/client-go/image/clientset/versioned"
```

Comment:
will need to check how much do we need from image client... if just Image then sure we can strip the client scope

Reply:
a) the import name doesn't match
b) I think unless you need an api function the client is the same just restricted to v1 instead of giving you the option to choose between internal and v1 as part of its interface